docs/edgeql/sdl/index.rst

```diff
@@ -13,13 +13,19 @@ language and the imperative low-level :ref:`DDL <ref_eql_ddl>`.
 
 SDL is a declarative language optimized for human readability and
 expressing the state of the EdgeDB schema without getting into the
-details of how to arrive at that state.  Each *.esdl* file represents
-a complete schema state for a particular
-:ref:`module <ref_datamodel_modules>`.
+details of how to arrive at that state.  Each *SDL* block represents
+the complete schema state for a given :ref:`database
+<ref_datamodel_databases>`.
 
 Syntactically, an SDL declaration mirrors the ``CREATE`` DDL for the
 corresponding entity, but with all of the ``CREATE`` and ``SET``
-keywords omitted.
+keywords omitted.  These individual declarations must either use
+:ref:`fully-qualified names <ref_eql_fundamentals_name_resolution>` or
+can be enclosed into a *module block* where only short names are
+allowed at top-level and any short name is assumed to belong either to
+the :ref:`module <ref_datamodel_modules>` named by the block or to
```

Comment:
This is one long sentence. Please break it into 2-3 sentences and add an example.

Reply:
OK, I'll try re-wording it and maybe moving it after the main canonical example. I'll add it in the form "this is an alternative way to do things" with its own example equivalent to the canonical.